pokete_classes/npcs.py

```diff
@@ -213,13 +209,13 @@ def action(self):
         if self.poke.hp > 0 and (self.name not in self.fig.used_npcs
                                  or not settings("save_trainers").val) \
                 and self.check_walk(self.fig.x, self.fig.y):
-            self.mvmp.full_show()
+            mvp.movemap.full_show()
             time.sleep(0.7)
             self.exclamate()
             self.walk_point(self.fig.x, self.fig.y)
             if any(poke.hp > 0 for poke in self.fig.pokes[:6]):
                 self.text(self.texts)
-                winner = self.fight([poke for poke in self.fig.pokes[:6]
+                winner = fm.fight([poke for poke in self.fig.pokes[:6]
                                     if poke.hp > 0][0],
                                     self.poke,
                                     info={"type": "duel", "player": self})
```

Comment:
Due to the change in line 218, these lines need to be unindented by two spaces:

```suggestion
                                   if poke.hp > 0][0],
                                  self.poke,
                                  info={"type": "duel", "player": self})
```